File: src/main/java/ua/softgroup/matrix/server/supervisor/consumer/json/CurrenciesJson.java
package ua.softgroup.matrix.server.supervisor.consumer.json;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class CurrenciesJson {

    @JsonProperty
    private boolean success;
    @JsonProperty
    private List<CurrencyJson> list;

    public boolean isSuccess() {
        return success;
    }

    public void setSuccess(boolean success) {
        this.success = success;
    }

    public List<CurrencyJson> getList() {
        return list;
    }

    public void setList(List<CurrencyJson> list) {
        this.list = list;
    }

    @Override
    public String toString() {
        return "CurrenciesJson{" +
                "success=" + success +
                ", list=" + list +
                '}';
    }
}

File: src/main/scala/ua/softgroup/matrix/server/supervisor/consumer/json/CurrenciesJson.scala
package ua.softgroup.matrix.server.supervisor.consumer.json

import java.util

import com.fasterxml.jackson.annotation.JsonProperty

import scala.beans.{BeanProperty, BooleanBeanProperty}

/**
  * @author Oleksandr Tyshkovets <olexandr.tyshkovets@gmail.com>
  */
class CurrenciesJson {

  @JsonProperty
  @BooleanBeanProperty
  var success: Boolean = _

  @JsonProperty
  @BeanProperty
  var list: util.List[CurrencyJson] = _

  override def toString = s"CurrenciesJson(success=$success, list=$list)"

}
